docs/apidocs.html

```diff
@@ -0,0 +1,33 @@
+<html>
+<head>
+  <meta charset="utf-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
+  <title>LoopBack API Documentation (local)</title>
+  <link rel="stylesheet" href="http://apidocs.loopback.io/css/bootstrap.min.css">
+  <link rel="stylesheet" href="http://apidocs.loopback.io/css/code-themes/sl-theme.css">
+  <link rel="stylesheet" href="http://apidocs.loopback.io/css/main.css">
+</head>
+<body>
+  <div class="col-lg-9 reset-home-content">
+    <div id="LoopBack4">
+      <h2>LoopBack4 packages (local)</h2>
+
+      <p>Run the following command to (re)build API doc files:
+      <p><pre><code>$ lerna run build:apidocs</code></pre>
```

Comment:
We probably should use `npx lerna run build:apidocs` or add a script to top level package.json.

Reply:
Good catch! I have `./node_modules/.bin` in my PATH and my Bash configured to run `npx` for commands not found, but I agree that's not something we can rely on in our docs.